system/modules/core/dca/tl_files.php

```diff
@@ -146,7 +146,8 @@
 		),
 		'path' => array
 		(
-			'sql'                     => "varchar(1022) NOT NULL default ''"
+			'eval'                    => array('unique'=>true),
```

Comment:
"unique" in terms of "unique for this installation". Everywhere we use `findByPath()` we expect a model, not a collection (if it was not unique it would return a collection, right?).
See e.g.
https://github.com/contao/core/blob/develop/system/modules/core/controllers/BackendPopup.php#L96
https://github.com/contao/core/blob/develop/system/modules/core/forms/FormFileUpload.php#L253
https://github.com/contao/core/blob/develop/system/modules/core/library/Contao/Picture.php#L123
https://github.com/contao/core/blob/develop/system/modules/core/modules/ModuleRegistration.php#L431